modules/tfm/zephyr/Kconfig

```diff
@@ -4,6 +4,8 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+menu "TFM"
```

Comment:
```suggestion
menu "TF-M"
```
Sorry

Reply:
Thanks, no problem; have a question though.
There are now two places with Kconfigs related to TF-M and I wonder if we could move these to one place. 

The top level `Kconfig.nrf` pulls `rsource "modules/Kconfig"`, and that in turns pulls `rsource "tfm/zephyr/Kconfig"`, with this menu.

But TF-M also has another menu containing all the Kconfig in `nrf/modules/trusted-firmware-m` and that menu is accessible in the `Modules` menu (beside `nrf` and all the other modules).

Reply:
There may be some issues with that, because that may change the evaluation order and cause issues.
Initially in was only TFM_BOARD config in this file so I think the rest can possible be moved. But moving TFM_BOARD could potentially be a problem. It's worth a shot if you want to try it though.